External/CoordGen/CoordGen.h

```diff
@@ -60,8 +63,9 @@ unsigned int addCoords(T& mol, const CoordGenParams* params = nullptr) {
 
   // FIX: only do this check once.
   std::cerr << "  TEMPLATES: " << templateFileDir << std::endl;
-  minimizer.setTemplateFileDir(templateFileDir);
-
+  if (templateFileDir != "") {
+    minimizer.setTemplateFileDir(templateFileDir);
```

Comment:
Is there any utility to ensure that the template file exists and throw an exception if not?

Reply:
Given that we (==RDKit) don't "own" the file, we'd be making an assumption, possibly an incorrect one, about what that file name is.

Reply:
Or did you mind check that the directory exists?

Reply:
The latter.  I'm assuming if the templatefileDir doesn't exist then bad things will happen.

Reply:
I just checked this on my windows machine and everything seems fine.
That's fortunate since `std::filesystem` is a C++17 thing and I'd rather not add the `boost::filesystem` dependency here.
What I will do is add a test to make sure that the coordgen stuff still works when a bogus directory name is provided.

Reply:
ok, that's now in